_includes/default-header.html

```diff
@@ -48,7 +48,7 @@
           <div class="navbar-item has-dropdown is-hoverable">
             <a class="navbar-link" href="/about">Policies</a>
             <div class="navbar-dropdown"> 
-              <a class="navbar-item" href="/code-of-conduct"> Code of Conduct </a>
+              <a class="navbar-item" href="https://github.com/open-life-science/policies-procedures-and-docs/blob/c342530ffe943bcc1a84a1a8a8e3b5b3c0fa4547/CODE_OF_CONDUCT.md"> Code of Conduct </a>
```

Comment:
Could you have the link the main branch there?

Reply:
What do you mean?
I don't understand....
The link is the link of the code-of-conduct file which is in the **main branch**
[This is it](https://github.com/open-life-science/policies-procedures-and-docs/blob/main/CODE_OF_CONDUCT.md)

Reply:
the link you put in the HTML file has a commit id instead of main after `blob` and before `CODE_OF_CONDUCT`

Reply:
Oh! really?
I'm correcting it now ✊

Reply:
@bebatut I already corrected it 🎉
Thanks for reviewing